search/views.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.generics import ListAPIView
+
+from profiles.models import Profile
+from .serializers import CompanySerializers
+
+
+class SearchCompanyView(ListAPIView):
+    serializer_class = CompanySerializers
+
+    def get_queryset(self):
+        search_field = self.request.query_params.get(
+            "search_field", ""
+        ).strip()
```

Comment:
Don't we use Django Filters?

What if we'll have to do search by multiple fields, + user-controlles ordering?

Reply:
Thank you for feedback. Changed it.